scripts/setup_env.py

```diff
@@ -64,3 +65,8 @@
 
 if "ONNX_MODEL_PATH" in param:
     print('export ONNX_MODEL_PKG="{}"'.format(os.path.join(param["ONNX_MODEL_PATH"])))
+
+if param.get("ISOTROPIC", False):
+    print('export ISOTROPIC="--isotropic"')
+else:
+    print('export ISOTROPIC="--anisotropic"')
```

Comment:
I think the red circle with a minus sign in it here is an indication of some LF/CRLF error or inconsistency, see if you can make it disappear